### ISSUE TITLE
Missing aria-valuenow?

### ISSUE THREAD

**Issue by jcoyne** [blueimp/jQuery-File-Upload#3162]
I don't understand aria well, so I'm posing this as a question as much as a bug report.  Should this line https://github.com/blueimp/jQuery-File-Upload/blob/52301068abb71b95fb61031fdca9f5f607193e05/index.html#L107  have an `aria-valuenow` attribute?
